packages/core/src/App/Containers/AccountSignupModal/account-signup-modal.jsx

```diff
@@ -183,17 +186,19 @@ AccountSignupModal.propTypes = {
     is_loading    : PropTypes.bool,
     is_visible    : PropTypes.bool,
     onSignup      : PropTypes.func,
+    pushDataLayer : PropTypes.func,
     residence_list: PropTypes.arrayOf(PropTypes.object),
 };
 
 export default connect(
-    ({ ui, client }) => ({
+    ({ ui, client, gtm }) => ({
         is_visible              : ui.is_account_signup_modal_visible,
         toggleAccountSignupModal: ui.toggleAccountSignupModal,
         enableApp               : ui.enableApp,
         disableApp              : ui.disableApp,
         is_loading              : ui.is_loading,
         onSignup                : client.onSignup,
+        pushDataLayer           : gtm.pushDataLayer,
```

Comment:
please put this on the last line, we do it alphabetically by module, and `gtm` comes after `client`